### PR TITLE
Improve some Ayu highlight colours in Helm buffers

### PR DESCRIPTION
**Pull request by offbyone**
> Please follow these steps before submitting your PR
>
> - [ ] Include before and after screenshots if your change is visual
> - [ ] Please explain what the PR does and why
> - [ ] List and link any packages you are adding support for
> - [ ] Delete this template before submitting
>
> Thank you for contributing to our theme pack!

### CODE REVIEW

**Comment by offbyone** [comment]
### Light
image

### Mirage
image

File: themes/doom-ayu-light-theme.el [hunk 1]
```diff
@@ -186,6 +186,11 @@ determine the exact padding."
    (ivy-minibuffer-match-face-3 :foreground common-accent :weight 'bold)
    (ivy-minibuffer-match-face-4 :foreground common-accent :weight 'bold)
 
+   ;;;; helm
+   (helm-ff-dotted-directory :inherit 'syntax-comment)
```
Comment (hlissner) [inline themes/doom-ayu-light-theme.el:190]: What is syntax-comment? It doesn't exist in Emacs 28. Please don't inherit from faces that aren't guaranteed to exist when helm is loaded.

File: themes/doom-ayu-mirage-theme.el [hunk 2]
```diff
@@ -157,6 +157,8 @@ determine the exact padding."
     :inherit 'font-lock-comment-face
     :foreground doc-comments)
 
+   (cursor :background "DarkOrange1")
```
Comment (hlissner) [inline themes/doom-ayu-mirage-theme.el:160]: Please use one of our color variables, or use `doom-darken` or `doom-lighten` to adjust them.

File: themes/doom-ayu-mirage-theme.el [hunk 3]
```diff
@@ -188,6 +190,11 @@ determine the exact padding."
    (ivy-minibuffer-match-face-3 :foreground common-accent :weight 'bold)
    (ivy-minibuffer-match-face-4 :foreground common-accent :weight 'bold)
 
+   ;;;; helm
+   (helm-ff-dotted-directory :inherit 'syntax-comment)
```
Comment (hlissner) [inline themes/doom-ayu-mirage-theme.el:194]: Again, `syntax-comment`

File: themes/doom-ayu-mirage-theme.el [hunk 4]
```diff
@@ -188,6 +190,11 @@ determine the exact padding."
...
+   (helm-ff-executable :inhert 'helm-ff-socket :weight 'bold)
+   (helm-selection :inherit 'normal :background "DarkOrange1")
```
Comment (hlissner) [inline themes/doom-ayu-mirage-theme.el:196]: Again, use a color variable.

File: themes/doom-ayu-light-theme.el [hunk 5]
```diff
@@ -186,6 +186,11 @@ determine the exact padding."
...
+   (helm-ff-executable :inhert 'helm-ff-socket :weight 'bold)
+   (helm-selection :inherit 'normal :background "#399ee6")
```
Comment (hlissner) [inline themes/doom-ayu-light-theme.el:192]: Please don't hardcode colors. Use a color variable and use doom-darken or doom-lighten to adjust them.

**Review by hlissner — CHANGES_REQUESTED** [review]
Could you include "before" screenshots? This PR does not describe how this is an improvement.

**Pull request closed without merging**